scripts/lib/install

```diff
@@ -100,8 +100,8 @@ export LANGUAGE="en_US.UTF-8"
 
 # Check for a supported OS release.
 if [ -f /etc/os-release ]; then
-    os_info="$(. /etc/os-release; printf '%s\n' "$ID" "$ID_LIKE" "$VERSION_ID" "$VERSION_CODENAME")"
-    { read -r os_id; read -r os_id_like; read -r os_version_id; read -r os_version_codename; } <<< "$os_info"
+    os_info="$(. /etc/os-release; printf '%s\n' "$ID" "$ID_LIKE" "$VERSION_ID")"
+    { read -r os_id; read -r os_id_like; read -r os_version_id; } <<< "$os_info"
```

Comment:
It’s fine to read an empty `VERSION_CODENAME` here. The only issue is that `$()` strips trailing newlines, causing an EOF on the last `read`. It’s probably simplest to ignore it:

```bash
    os_info="$(. /etc/os-release; printf '%s\n' "$ID" "$ID_LIKE" "$VERSION_ID" "$VERSION_CODENAME")"
    { read -r os_id; read -r os_id_like; read -r os_version_id; read -r os_version_codename || true; } <<< "$os_info"
```

Reply:
Fixed.